lib/atlas/dataset/derived_dataset.rb

```diff
@@ -4,7 +4,7 @@ class Dataset::DerivedDataset < Dataset
 
     attribute :base_dataset, String
     attribute :scaling,      Preset::Scaling
-    attribute :init,         Hash[Symbol => Float]
+    attribute :init,         Hash[InitializerInput => Float]
```

Comment:
> Would that be done in ETEngine?

I'm not sure; I'm in two minds about this. Do you have a preference?

An alternative is to define an `initializer_inputs` method on `DerivedDataset`: `init` is the dumb hash, `initializer_inputs` returns a map of inputs and values. Does that feel "clunky" and awkward to you? It doesn't feel quite right to me, even though there is some precedent  with methods like `Node#(in|out)_slots`.

I think my comparison with Preset was not very fair, since it doesn't *have* to map keys to inputs: `user_values` just get saved straight to the DB when a scenario uses the preset.

In hindsight, I think I'm okay with `{ InitializerInput => Float }` provided the document still loads when an input doesn't exist, and that it can be validated the same as Preset.